packages/ListBoxBrowser/src/components/CustomListBox/CustomListBox.js

```diff
@@ -60,7 +60,7 @@ export default function CustomListBox(props) {
       <ListBox.Trigger isHidden />
       {hasOnUp ? (
         <ListBox.Option onClick={onUp}>
-          <span css={backButton} data-pka-anchor="listboxbrowser-listoption-back">
+          <span css={backButton} data-pka-anchor="list-box-browser-listoption-back">
```

Comment:
While we're making breaking changes, let's adjust these anchors to follow the [naming conventions](https://github.com/acl-services/paprika/wiki/Coding-Conventions#anchors) for them, so in this case, `list-box-browser.list-option-back`.